client/websocket_client.jsx

```diff
@@ -2,7 +2,7 @@
 // See LICENSE.txt for license information.
 
 const MAX_WEBSOCKET_FAILS = 7;
-const MIN_WEBSOCKET_RETRY_TIME = 3000; // 3 sec
+const MIN_WEBSOCKET_RETRY_TIME = 30000; // 3 sec
```

Comment:
Nit: 30 sec? :P

Reply:
Oops, It was a value left from testing socket disconnect scenario with PM. Removed it